### PR TITLE
Adding Code of Conduct

### PR DESCRIPTION
**Pull request by truongnh1992**
Signed-off-by: Nguyen Hai Truong <truongnh@fujitsu.com>

Please fill in the fields below to submit a pull request.  The more information
that is provided, the better.

**Fixes issue #**:

**Description of the changes being introduced by the pull request**:

**Please verify and check that the pull request fulfills the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Tuf Community Code of Conduct
```
Comment (JustinCappos) [inline CODE_OF_CONDUCT.md]:
```suggestion
## TUF Community Code of Conduct
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+Tuf follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```
Comment (JustinCappos) [inline CODE_OF_CONDUCT.md]:
```suggestion
TUF follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

**Review by JustinCappos — COMMENTED** [review]
We have this file in our docs/ directory already.  Is this not sufficient?

**Comment by truongnh1992** [comment]
Thank @JustinCappos for your comment. I will close the PR.

**Pull request closed without merging**